index.js

```diff
@@ -132,12 +132,14 @@ function runNightwatch (done) {
 				 */
 				function (cb) {
 					Nightwatch.runner(argv, function (status) {
+						var err = null;
 						if (status) {
 							console.log([moment().format('HH:mm:ss:SSS')] + ' kne: tests passed');
 						} else {
 							console.log([moment().format('HH:mm:ss:SSS')] + ' kne: tests failed');
+							err = new Error('kne: nightwatch runner return error status code: 0');
```

Comment:
Let me research this more later today.  But I think nightwatch does the reverse of unix with status codes :=)